python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -92,15 +128,15 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
         actor.pid,
         duration,
       );
-      this.setState((state) => ({
+      this.setState(state => ({
```

Comment:
Please make sure you run `npm install` to get the latest version of Prettier to avoid this reverted style change.